### ISSUE TITLE
Update to FixedEffectModels 0.8.2

### ISSUE THREAD

**Issue by matthieugomez** [jmboehm/RegressionTables.jl#27]
The type `AbstractRegressionResult` is not longer defined in FixedEffectModel 0.8.2. 
`reg` now returns a `FixedEffectModel` that inherits from `RegressionModel`.

I think the main thing you need to do is to update the code [here](https://github.com/jmboehm/RegressionTables.jl/blob/ed80aafad231218ace2ec577d18b2cec1bc34211/src/util/util.jl), using the newly exported functions `has_iv` and `has_fe`:

```julia
isFERegressionResult(r::FixedEffectModel) = has_fe(r)
isIVRegressionResult(r::FixedEffectModel) = has_iv(r)
isOLSRegressionResult(r::FixedEffectModel) = !has_iv(r)
```

**Comment by jlperla** [comment]
@matthieugomez @jmboehm  Can you guys tell me (and @arnavs) when you think a good time to bump the fixedeffectmodels+regressiontables ecosystem for the quantecon lectures?

If you are curious, these two packages are (usually) in https://lectures.quantecon.org/jl/more_julia/data_statistical_packages.html  but we had to comment it out due to https://github.com/jmboehm/RegressionTables.jl/issues/26 but I tested those fixes.

**Comment by jmboehm** [comment]
@jlperla @arnavs This should be solved by #29 . I've opened the PR for a new version to be registered. Once that is merged, I think you can put it back into quantecon.

**Comment by arnavs** [comment]
Perfect, thank you @jmboehm. Looking forward to getting this package back in the notes!
